satellite/satellitedb/overlaycache.go

```diff
@@ -1080,6 +1097,17 @@ func convertDBNode(ctx context.Context, info *dbx.Node) (_ *overlay.NodeDossier,
 		Patch: info.Patch,
 	}
 
+	exitStatus := overlay.ExitStatus{NodeID: id}
+	if info.ExitInitiatedAt != nil {
```

Comment:
nit - I don't think these if statements are necessary. If `info.ExitInitiatedAt == nil` it should still be fine to do `exitStatus.ExitInitiatedAt = info.ExitInitiatedAt`.

Reply:
Also, shouldn't `ExitInitiatedAt` be a normal timestamp and not a pointer since it cannot be `NULL` in sql?

Reply:
Comment 1: Agreed. Updated
Comment 2: `exit_initiated_at` can be NULL